iothub/service/src/DirectMethod/Models/DirectMethodServiceRequest.cs

```diff
@@ -59,35 +63,19 @@ public DirectMethodServiceRequest()
         public TimeSpan? ResponseTimeout { get; set; }
 
         /// <summary>
-        /// Get the payload object. May be null or empty.
-        /// </summary>
-        [JsonIgnore]
-        public object Payload
-        {
-            get => _payload;
-
-            set
-            {
-                _payload = value;
-                if (value != null)
-                {
-                    PayloadAsJsonString = JsonSerializer.Serialize(value);
-                    JsonPayload = new JRaw(PayloadAsJsonString);
-                }
-            }
-        }
-
-        /// <summary>
-        /// Get the serialized JSON payload. May be null or empty.
+        /// Get the serialized JSON payload.
         /// </summary>
+        /// <remarks>
+        /// May be null or empty.
+        /// </remarks>
         [JsonIgnore]
-        public string PayloadAsJsonString { get; internal set; }
+        public string PayloadAsJsonString => JsonPayload?.GetRawText();
 
         /// <summary>
-        /// The JSON payload in JRaw type.
+        /// The deserialized payload.
         /// </summary>
         [JsonPropertyName("payload")]
-        internal JRaw JsonPayload { get; set; }
+        internal JsonElement? JsonPayload { get; set; }
```

Comment:
Is this the property that eventually gets serialized and sent over in the transport layer?